app/services/nomis_api_client.rb

```diff
@@ -27,11 +24,15 @@ def get_version_timestamp
   end
 
   def get(path)
-    NOMIS::API::Get.new(
-      client_key: client_key,
-      client_token: client_token,
-      base_url: base_url,
-      path: path
-    ).execute
+    begin
+      parser.parse(NOMIS::API::Get.new(
+        client_key: client_key,
+        client_token: client_token,
+        base_url: base_url,
+        path: path
+      ).execute)
+    rescue => exception
+      parser.parse(exception)
```

Comment:
by this point we should know that it's an ExceptionSafeThing so I have used a shorter instance var